index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    
+    <header>
+        <div class="logo">Vishal</div>
```

Comment:
- [ ] God job adding your logo but I will strongly recommend that you make your logo clickable by wrapping it in an anchor tag `<a>` linking to the homepage so that users can be able to navigate with ease which is a good user experience

e.g 

`<a href="/" class="logo">Vishal</a>`